src/Tizen.Multimedia.Remoting/WebRTC/WebRTCDataChannel.Events.cs

```diff
@@ -132,6 +134,31 @@ public partial class WebRTCDataChannel
             }
         }
 
+        /// <summary>
+        /// Occurs when the buffered amount is lower than <see cref="BufferedAmountLowThreshold"/>.<br/>
+        /// If <see cref="BufferedAmountLowThreshold"/> is not set, this event will not occurred.
```

Comment:
"will not occurred"  -> will not be raised / will not be triggered / will not occur 
```suggestion
        /// Occurs when the buffered data amount is lower than <see cref="BufferedAmountLowThreshold"/>.<br/>
        /// If <see cref="BufferedAmountLowThreshold"/> is not set, this event will not be raised.
```

Reply:
Okay. I'll fix it.